src/core/hle/service/glue/time/time_zone.cpp

```diff
@@ -0,0 +1,373 @@
+// SPDX-FileCopyrightText: Copyright 2023 yuzu Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include <chrono>
+
+#include "core/core.h"
+#include "core/hle/kernel/svc.h"
+#include "core/hle/service/glue/time/file_timestamp_worker.h"
+#include "core/hle/service/glue/time/time_zone.h"
+#include "core/hle/service/glue/time/time_zone_binary.h"
+#include "core/hle/service/psc/time/time_zone_service.h"
+#include "core/hle/service/set/set_sys.h"
+#include "core/hle/service/sm/sm.h"
+
+namespace Service::Glue::Time {
+namespace {
+static std::mutex s_list_mutex;
+static Common::IntrusiveListBaseTraits<Service::PSC::Time::OperationEvent>::ListType s_list_nodes{};
```

Comment:
```suggestion
static std::mutex g_list_mutex;
static Common::IntrusiveListBaseTraits<Service::PSC::Time::OperationEvent>::ListType g_list_nodes{};
```
These are globals, not static class members. Also, it would be better if you didn't have globals, and tied these to the lifetime of a container created in the LoopProcess for glue (see [here](https://github.com/yuzu-emu/yuzu/blob/master/src/core/hle/service/ro/ro.cpp#L691) for an example)

Reply:
They're static to the file, don't see why they have to be inside a class for that.